tests/Unit/Service/WpCronServiceTest.php

```diff
@@ -62,3 +64,59 @@
         ->and($firstTask['args'])
         ->toBe(['arg1', 'arg2']);
 });
+
+it('can call a callable', function () {
+    /** @var \MyParcelNL\WooCommerce\Tests\Mock\WordPressScheduledTasks $tasks */
+    $tasks = Pdk::get(WordPressScheduledTasks::class);
+    /** @var \MyParcelNL\Pdk\Base\Contract\CronServiceInterface $cronService */
+    $cronService = Pdk::get(CronServiceInterface::class);
+
+    $mockClass = new MockClass();
+
+    $cronService->dispatch([$mockClass, 'mock'], 'arg', true);
+    $cronService->dispatch('MyParcelNL\WooCommerce\Tests\Mock\MockClass::mock', 'arg', true);
+
+    $firstTask = $tasks
+        ->all()
+        ->first();
+
+    $lastTask = $tasks
+        ->all()
+        ->last();
+
+    $actions = get_option(Pdk::get('myParcelActions'));
+
+    expect($tasks->all())
+        ->toHaveLength(2)
+        ->and($firstTask['callback'])
+        ->toBeString()
+        ->and($firstTask['args'])
+        ->toBe(['arg', true])
+        ->and(
+            $actions[sprintf(
+                '%s%s',
+                Pdk::get('myParcelHook'),
```

Comment:
hier ook weer ff normaal concatenaten, en zet het meteen in een variabele want je doet het hieronder weer